### PR TITLE
Update CODE_OF_CONDUCT.md

### LINKED ISSUE #4: Code of Conduct update - need contacts for conflict of interest
https://github.com/ProtoSchool/Denver/blob/master/CODE_OF_CONDUCT.md

Updated with contact info link to the README.md

@terichadbourne  - I have you listed in case of a conflict of interest, but who else should be listed? And if for some crazy reason we would need to escalate to another person/party at protocol.ai - who would the person be? 

(Or am I getting to deep into this :sweat_smile: )

### PR DESCRIPTION
**Pull request by nuke-web3**
To fix #4 

@terichadbourne  - look good on your end? Thanks for the better content!

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -58,11 +58,7 @@ If you believe someone has violated our Code of Conduct, we encourage you to rep
 
 Make a report to any of the maintainers listed in the ProtoSchool Denver Chapter [README.md](https://github.com/ProtoSchool/Denver/blob/master/README.md):
  
-**In the event of a conflict of interest, you are encouraged to directly contact any of other ProtoSchool Chaper Maintainers, [found here](https://github.com/ProtoSchool), or the Orginization's lead maintainers:**
-
-- [Teri Chadbourne](https://terichadbourne.com/)
-Community Manager at [ProtocolLabs](https://protocol.ai)
-[teri@protocol.ai](teri@protocol.ai)
+**In case of a conflict of interest with the Denver Chaper maintainers, or if you feel that this chapter's organizers are not living up to the standards laid out in our Code of Conduct, please follow the reporting procedures laid out in the broader [ProtoSchool Code of Conduct](https://github.com/ProtoSchool/organizing/blob/master/CODE_OF_CONDUCT.md) to reach the project's lead maintainers - [protoschool@protocol.ai](protoschool@protocol.ai)**
```
Comment (terichadbourne) [inline CODE_OF_CONDUCT.md]: "Denver Chaper" -> "Denver Chapter"

Comment (terichadbourne) [reply CODE_OF_CONDUCT.md]: The link created for protoschool@protocol.ai tries to go to https://github.com/ProtoSchool/Denver/blob/96cbf3cbf8391066e141f72a53ca7acf0da1cd6a/protoschool@protocol.ai and gets a 404. I presume you need to prepend `mailto:`.

**Comment by terichadbourne** [comment]
Looks good apart from the two typos noted. Thanks!

**Comment by nuke-web3** [comment]
Thanks for the typo catches! :grin:

**Pull request merged**